### PR TITLE
Use steps instead of installer template

### PR DESCRIPTION
**Pull request by dhiller**
Check whether we can quickly change our setup to steps.

### CODE REVIEW

**Comment by fabiand** [comment]
That was quick!

I'm staying tuned.

**Comment by dhiller** [comment]
Looking good so far, but test step is failing due to missing `oc` binary.

File: ci-operator/config/kubevirt/kubevirt/kubevirt-kubevirt-master.yaml [hunk 1]
```diff
@@ -38,6 +38,23 @@ tests:
       commands: |
         set -euo pipefail
 
+        # FIXME: transfer into Dockerfile for ci
```
Comment (wainersm) [inline ci-operator/config/kubevirt/kubevirt/kubevirt-kubevirt-master.yaml:41]: Mind to ask one thing @dhiller, aren't the oc and kubectl already present in the shared folder (${SHARED_DIR}) ?

Comment (dhiller) [reply ci-operator/config/kubevirt/kubevirt/kubevirt-kubevirt-master.yaml:41]: No they aren't any more, that's one thing everyone now needs to provide on his own, AFAIU DPTP is working on a solution for this.

Comment (wainersm) [reply ci-operator/config/kubevirt/kubevirt/kubevirt-kubevirt-master.yaml:41]: ah, ok, so I know my current job with changes for multistage test steps is going to fail :)

**Pull request closed without merging**